modules/virt-accessing-vmi-ssh.adoc

```diff
@@ -4,51 +4,82 @@
 
 :_content-type: PROCEDURE
 [id="virt-accessing-vmi-ssh_{context}"]
-= Accessing a virtual machine instance via SSH
+= Accessing a virtual machine via SSH by using virtctl
 
-You can use SSH to access a virtual machine (VM) after you expose port 22 on it.
+You can use `virtctl` to forward SSH traffic to a virtual machine (VM). The `virtctl ssh` command allows you to establish SSH connections to VMs by using your local SSH client.
 
-The `virtctl expose` command forwards a virtual machine instance (VMI) port to a node
-port and creates a service for enabled access. The following example creates
-the `fedora-vm-ssh` service that forwards traffic from a specific port of cluster nodes to port 22 of the `<fedora-vm>` virtual
-machine.
+[NOTE]
+====
+Heavy SSH traffic on the control plane can slow down the API server. If you regularly need a high amount of connections and traffic, use a dedicated Kubernetes `Service` object to access the virtual machine.
+====
 
 .Prerequisites
-* You must be in the same project as the VMI.
-* The VMI you want to access must be connected
-to the default pod network by using the `masquerade` binding method.
-* The VMI you want to access must be running.
-* Install the OpenShift CLI (`oc`).
+* You have access to an {product-title} cluster with `cluster-admin` permissions.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `virtctl` client.
+* The virtual machine you want to access is running.
+
 
 .Procedure
-. Run the following command to create the `fedora-vm-ssh` service:
+
+. Use the `ssh-keygen` command to generate an SSH public key pair:
 +
+[source,terminal]
 ----
-$ virtctl expose vm <fedora-vm> --port=22 --name=fedora-vm-ssh --type=NodePort <1>
+$ ssh-keygen -f id_rsa
```

Comment:
`id_rsa` is also the default for the user's identity file. To make it clearer that this is a custom key created by the user, naming it differently might make sense. E.g. `ssh-keygen -f my-key`?

Reply:
@0xFelix So if I replace `id_rsa` with `my-key` here, then I am guessing I would need to replace all instances of `id_rsa` in the other commands as well, correct?

Reply:
Correct!